src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static void printLine(){
+        System.out.println("--------------------------------------------------");
+    }
+
+    public static void greet(){
+        printLine();
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+        printLine();
+    }
+
+    public static void echo(String line){
+        printLine();
+        System.out.println(line);
+        printLine();
+    }
+
+    public static void bye(){
+        printLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        printLine();
+    }
+
+    public static void acknowledgement(Task message, int i){
+        System.out.println("Got it. I've added this task:");
+        System.out.println("    " + message);
+        System.out.println("Now you have " + i + (i == 1 ? " task" : " tasks") + " in your list.");
+    }
+
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
```

Comment:
Try using more explicit variable names so that readers can identify them in the code better